### PR TITLE
auth is working, however there is a large firebase lag.…

### PR DESCRIPTION
**Pull request by isaacblinder**
… Initial project structure created for homepage, profile and songProfile pages, as well as a router to handle requests.
The lag, which I've narrowed down to our firebase connection, causes a large delay to displaying content, logging in / out users and signup. I'm also having issues with yarn not responding once I make changes. Please let me know if you run into these same issues so I can tell whether this is a problem with my own setup or not.
Also be sure to run 'npm install' as there are some new dependencies.

### CODE REVIEW

**Review by ann0605 — APPROVED** [review]

**Comment by ann0605** [comment]
wait can you do a PR against /development branch ? :)

**Comment by isaacblinder** [comment]
> wait can you do a PR against /development branch ? :)

Is there a way to change the destination or should I make another PR?

File: src/store/actions/authActions.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    const firebase = getFirebase();
+    const firestore = getFirestore();
+
+    firebase.auth().createUserWithEmailAndPassword(
+        newUser.email,
+        newUser.password
+    ).then((resp)=> {
+        return firestore.collection('users').doc(resp.user.uid).set({
+            fullName: newUser.fullName,
+            photoUrl: '',
```
Comment (ann0605) [inline src/store/actions/authActions.js:40]: also do you mind adding artistName and changing "fullName" to accountOwner to match the models ?

**Comment by ann0605** [comment]
I had to close and create a new PR against the development branch; not sure if there is a better way

File: src/store/actions/authActions.js (hunk 1 above)
Comment (isaacblinder) [reply src/store/actions/authActions.js:40]: Okay. I'll add and create another pull request

**Pull request closed without merging**